### ISSUE TITLE
Cannot build Android App on Bitrise

### ISSUE THREAD

**Issue by felixjunghans** [simolus3/moor#1160]
Hi, 

when I try to build my Android App on Bitrise I got the following error: 

```
Could not determine the dependencies of task ':app:collectLogBuchReleaseDependencies'.
> Could not resolve all task dependencies for configuration ':app:logBuchReleaseRuntimeClasspath'.
   > Could not resolve eu.simonbinder:sqlite3-native-library:3.32.3.
     Required by:
         project :app > project :sqlite3_flutter_libs
      > Could not resolve eu.simonbinder:sqlite3-native-library:3.32.3.
         > Could not parse POM https://bintray.com/larpoux/CanardouxMaven/eu/simonbinder/sqlite3-native-library/3.32.3/sqlite3-native-library-3.32.3.pom
            > The markup in the document preceding the root element must be well-formed.
      > Could not resolve eu.simonbinder:sqlite3-native-library:3.32.3.
         > Could not get resource 'https://google.bintray.com/exoplayer/eu/simonbinder/sqlite3-native-library/3.32.3/sqlite3-native-library-3.32.3.pom'.
            > Could not HEAD 'https://google.bintray.com/exoplayer/eu/simonbinder/sqlite3-native-library/3.32.3/sqlite3-native-library-3.32.3.pom'. Received status code 403 from server: Forbidden
      > Could not resolve eu.simonbinder:sqlite3-native-library:3.32.3.
         > Could not get resource 'https://dl.bintray.com/sbinder/sqlite3-native-library/eu/simonbinder/sqlite3-native-library/3.32.3/sqlite3-native-library-3.32.3.pom'.
            > Could not HEAD 'https://dl.bintray.com/sbinder/sqlite3-native-library/eu/simonbinder/sqlite3-native-library/3.32.3/sqlite3-native-library-3.32.3.pom'. Received status code 403 from server: Forbidden
```

This does not happend before (Last build on Friday). 
My versions of moor are: 

  moor: 3.1.0
  moor_ffi: ^0.8.0
  
  Can you help me with this?

**Comment by kuhnroyal** [comment]
I think you may need to update moor and/or moor_ffi.
Bintray is no longer. See https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

It may work locally due to cached dependencies.

You can also try adding a newer version of https://pub.dev/packages/sqlite3_flutter_libs directly.

This was changed in https://github.com/simolus3/sqlite3.dart/commit/2b0c1ffa9e869144680e4b31914d7c81d7044397#diff-30bdf6c5328f273f91b808d80d0401195a367700e0854c7363956a2c4e553182

**Comment by felixjunghans** [comment]
Ok. Thanks for your quick feedback :)

**Comment by simolus3** [comment]
Note that the change to `simonbinder.eu/maven` was temporary (never published to pub) and no longer works. But `sqlite3-native-library` is on maven central now, so if you upgrade your version of `sqlite3_flutter_libs` (use `0.4.0+1` or later) it should work.

**Comment by jamie1192** [comment]
@simolus3 , is it possible to publish a version that of `sqlite3_flutter_libs` that fixes this issue but doesn't require Dart > 2.12? We have an app from last year that we are unable to migrate to null safety yet, but need to publish an update for, and can no longer do so through CI due to this issue :/

**Comment by simolus3** [comment]
Please note that you can use Flutter 2/Dart 2.12 _without_ migrating your app to null safety, just keep something like this in your pubspec:

```yaml
environment:
  sdk: '>=2.11.0 <3.0.0'
```

Regardless, version `0.4.0+1` of `sqlite3_flutter_libs`  uses the new libraries from Maven Central without having a tight lower sdk constraint.
